plugins/modules/ec2_vpc_route_table_info.py

```diff
@@ -59,20 +59,35 @@
   type: complex
   contains:
     associations:
-      description: List of subnets associated with the route table
+      description: List of associations between the route table and one or more subnets or a gateway
       returned: always
       type: complex
       contains:
+        association_state:
+          description: The state of the association
```

Comment:
```suggestion
          description: The state of the association.
```

Reply:
@Akasurde so I left periods off ends of sentences in the return block because I noticed they were only included in descriptions in the documentation section

Reply:
As far as I know, we need to have the periods at the end of the sentences even in the RETURN block.